test/parallel/test-buffer-arraybuffer.js

```diff
@@ -43,12 +43,16 @@ assert.throws(function() {
   Buffer.from(new AB());
 }, TypeError);
 
-// write{Double,Float}{LE,BE} with noAssert should not crash, cf. #3766
+// https://github.com/nodejs/node/issues/3766
 const b = Buffer.allocUnsafe(1);
-b.writeFloatLE(11.11, 0, true);
-b.writeFloatBE(11.11, 0, true);
-b.writeDoubleLE(11.11, 0, true);
-b.writeDoubleBE(11.11, 0, true);
+assert.throws(() => b.writeFloatLE(11.11, 0),
+              /RangeError: out of range index/);
+assert.throws(() => b.writeFloatBE(11.11, 0),
+              /RangeError: out of range index/);
```

Comment:
If you decide to keep these four tests, though, please add a `^` and `$` to the start and end respectively of the RegExp.